openedx/core/djangoapps/content/learning_sequences/data.py

```diff
@@ -213,7 +213,7 @@ def not_deprecated(self, _attribute, value):
     course_visibility: CourseVisibility = attr.ib(validator=attr.validators.in_(CourseVisibility))
 
     # Entrance Exam ID
-    entrance_exam_id = attr.ib(type=str)
+    entrance_exam_id = attr.ib(type=Optional[str])
```

Comment:
It was unclear if this should be changed to allow `None` or if `None` values should be converted to `""` when stored in this field, but there were clear examples if it being explicitly set to `None` in the tests, so I went with this.

https://github.com/openedx/edx-platform/blob/478c301df5ab0a58dad43cedfc05e4003828c2d3/openedx/core/djangoapps/content/learning_sequences/tests/test_views.py#L55